packages/f-searchbox/README.md

```diff
@@ -94,13 +97,55 @@ const options = {                 // (opt) component settings
   isCompressed: false,            // Boolean - hide/show compressed searchbox
   query: '',                      // String - override default query field value
   queryParams: {},                // Object - Query parameter overrides as key value pairs
-  onSubmit: a => void,            // Function - called when user selected a valid address, prevents form submit
+  onSubmit: a => void,            // Function - called when user selects a valid address, prevents f-searchbox from generating and submitting a custom post request and instead allows the consuming app to handle its own custom submit.
   shouldSetCookies: false,              // Boolean - sets je default location cookies
   shouldAutoPopulateAddress: true,      // Boolean - should the address value be auto-populated?
   shouldClearAddressOnValidSubmit: true // Boolean - should the address be cleared when a valid form is submitted?
 }
 ```
 
+## Configuration
+
+### Props
+
+`f-searchbox` has a number of props that allow you to customise its functionality.
+
+The props that can be defined are as follows:
+
+| Prop                      | Required       | Type          | Default | Description |
+| :---                      |     :---:      |     :---:     |  :---:  | :---        |
+| locale                    | false          | `string`      | `en-GB` | Sets the locale of the component (which determines what theme and translations to use |
+| config                    | false          | `object`      | `{}`    | When passed in `f-searchbox` will allow presentational & functional customisation (see options section) |
```

Comment:
Probably worth adding some detail at some point on what this object can contain.

Reply:
Do you mean add the `options` (further up) into this table?

Reply:
Ah, yes. Maybe that section can be updated at some point to be in the same table format?

Reply:
I think the fact the variable was labelled as "options" rather than "config" threw me.

Reply:
Yeah i just noticed that too. Ill update that.